### PR TITLE
Antifragility post

### PR DESCRIPTION
**Pull request by rhnonose**

### CODE REVIEW

File: _posts/2020-02-21-antifragile.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "AntifrAGILILITY"
```
Comment (gusaiani) [inline _posts/2020-02-21-antifragile.md:3]: @rhnonose hheheeheheh

File: _posts/2020-02-21-antifragile.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+date:   2020-02-21 19:00:00 -0300
+categories: agile
+---
+
+Recently, I gave a talk on [agile meetup][agile meetup] about something that's been bothering me for a while. This is a post-talk write-up that I intend to expand further.
+
+I've had mixed feeling about agile methodologies for a while and definitely a problem with the agile cargo culting.
```
Comment (gusaiani) [inline _posts/2020-02-21-antifragile.md]: s/mixed feeling/mixed feelings ?

File: _posts/2020-02-21-antifragile.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+I was never quite certain why since my day-to-day job is mostly technical and I had the luxury of not having to care.
+Until recently, for 2 years, I was in a semi-leadership position where I had to care.
+Upon reflecting on that experience and reading [antifragile][antifragile], I started making connections everywhere.
+My confirmation bias was making me think everything could be explained through this concept, and "agility" wasn't an exception.
+
+To gloss over a bit on the concept, `antifragility` is a recently coined term by [Nassim Nicholas Taleb][taleb] that describes a property of systems that gain from disorder, in contrast to the fragile that loses from disorder and the robust that doesn't care much.
+Complex systems such as life, communities and ecossystems can't be understood in full, but we can measure it's fragility by an assymetry test: if a system gains more from a stressor than it loses, it's antifragile.
```
Comment (gusaiani) [inline _posts/2020-02-21-antifragile.md]: s/it's fragility/its fragility/ ?

Comment (gusaiani) [reply _posts/2020-02-21-antifragile.md]: or rather `their fragilities` as it refers to life, communities and ecosystems?

Comment (gusaiani) [reply _posts/2020-02-21-antifragile.md]: s/ecossystems/ecosystem

Comment (gusaiani) [reply _posts/2020-02-21-antifragile.md]: s/assymetry/asymetry... yeah, a spellcheck run will fix these little things ¯\_(ツ)_/¯

File: _posts/2020-02-21-antifragile.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The waterfall method was very risky and fragilized the team/company since the project could be all wrong in the end of a cycle and invalidate all the work done, so the agile methodologies reduced the risk in smaller ones through iterative cycles and feedback.
+It introduced stressors to the systems such as recurring requirement changes - and encouraged the team to embrace it - and continuous feedback from client through collaboration over contract negotiation.
+It also recognized that, by letting individuals experiment and fail, it created a loop of constant trial, tinkering and error.
+
+A lot of the practices may make the system antifragile.
+Pair programming shares knowledge and creates redundancy (a push from fragile to robust), increasing the [bus factor][bus factor].
+The simplicity principle of reducing unnecessary work also contributes to a more predictable change in the system [via negativa][via negativa].
+It also gives autonomy to teams, reducing cascading errors that the entire organization suffers.
```
Comment (gusaiani) [inline _posts/2020-02-21-antifragile.md]: s/that the entire organization suffers/that makes the entire organization suffer ? or something like this?

File: _posts/2020-02-21-antifragile.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+From the beginning of the "movement" (when the [manifesto][agile manifesto] was written), the agile methodologies surfaced as a winner in midst of failed other methodologies, so the weaker counterparts had already died.
```
Comment (gusaiani) [inline _posts/2020-02-21-antifragile.md]: I think it is typically s/in midst of/in the midst of

**Review by gusaiani — COMMENTED** [review]
@rhnonose Cool post so far, and a very interesting connection between Antifragility and Agility.

We write, often, to learn and to organize your minds, seems like you are doing this one for this purpose. Meaning you probably haven't yet come out of this meditation with a new direction or point of view about Agility or collaboration methods in software. And man, that IS ok :) Just please share your conclusions with me as you draw them :D

Do you know this tool? It gives good grammar suggestions sometimes. https://www.grammarly.com/

**Comment by rhnonose** [comment]
Since I'm versioning my text, I found it difficult to use grammarly-like editors.
But there's too many typos 😅 I'll jump to that.

**Comment by gusaiani** [comment]
@rhnonose You could paste the text into some textarea in a browser just to trigger Grammarly if you have the browser extension installed (it's really helpful sometimes!). Also Grammarly has a native app (I've never used it though)

**Comment by rhnonose** [comment]
And also, from your conclusion part about the article, yes, it's for organizing my thoughts.
But I also don't intend to conclude anything yet. I'm thinking about picking individual methodologies and making an analysis from this point of view, which will force me to re-read some of the classics.

**Comment by gusaiani** [comment]
@rhnonose 🤙!

Have you read Jeff Sutherland's book, Twice the Work??

**Comment by rhnonose** [comment]
> @rhnonose 🤙!
> 
> Have you read Jeff Sutherland's book, Twice the Work??

I think not, I'll take a look.
For this topic, I picked mythical man-month and the cathedral and the bazaar.

**Comment by gusaiani** [comment]
I read part of MMM (felt it was okay, certainly makes sense), didn't know about the other one, thank you! Looks very interesting.

Added to https://www.goodreads.com/user/show/27282668-gustavo-saiani, hope you join Goodreads one day, are you there?

**Comment by rhnonose** [comment]
> Added to https://www.goodreads.com/user/show/27282668-gustavo-saiani, hope you join Goodreads one day, are you there?

I am now 🙂

**Comment by gusaiani** [comment]
@rhnonose \O/ 🇧🇷 🚀 :elixir:

**Pull request merged**